accounting/invoice/templates/default/default.html

```diff
@@ -0,0 +1,134 @@
+<!--
+  - OpenCraft -- tools to aid developing and hosting free software projects
+  - Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -
+-->
+
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="{{ site.domain }}">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="{% static 'vendor/css/bootstrap.min.css' %}"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'default/css/default.css' %}"/>
+  </head>
+  <body>
+    <div class="container">
+      <div class="invoice">
+        <header>
+          <div class="row">
+            <div class="col-xs-6">
+              <h1><strong>{{ invoice.provider }}</strong></h1>
+              {% if invoice.provider.occupation %}
+                <h4>{{ invoice.provider.occupation }}</h4>
+              {% endif %}
+            </div>
+            <div id="invoice-metadata" class="col-xs-6">
+              <span id="invoice-number">Invoice Number: {{ invoice.number }}</span><br>
+              <span id="invoice-created">Created: {{ invoice.date|date:"F d, Y" }}</span><br>
+              <span id="invoice-billing-dates">Billed for {{ invoice.billing_start_date|date:"F d, Y" }} to {{ invoice.billing_end_date|date:"F d, Y" }}</span>
+              <span id="invoice-due">Due: {{ invoice.due_date|date:"F d, Y" }}</span>
+            </div>
+          </div>
+          <div class="row">
+            {% for user_str, user in users %}
```

Comment:
@UmanShahzad What is `user_str`? Is there a more descriptive variable name that we could use here?

Reply:
I guess `user_type` would do. It's referring to the actual string representing what this is, because we style things slightly differently depending on whether it's `client-details` or `provider-details`.